subjects/java/checkpoints/spiral-matrix/README.md

```diff
@@ -0,0 +1,63 @@
+## Spiral Matrix
+
+### Instructions
+
+Create a class `SpiralMatrix` that provides a method to generate a matrix filled with elements from 1 to n\*n in spiral order, given an integer n.
```

Comment:
What should be the output for a negative number or 0?

Reply:
`n` is the length of the matrix and the length can't be negative, in case of length equal to 0 empty matrix should be returned

Reply:
@zanninso An `int` can be negative so possible is possible (even if it doesn't make sense :sweat_smile:)

I feel there should be either:
1 - a description of what's expected when the integer passed is negative or 
2 - an information that no negative numbers will be tested.

Reply:
done